ztools/x509/pkix/pkix.go

```diff
@@ -208,3 +208,17 @@ type RevokedCertificate struct {
 	RevocationTime time.Time
 	Extensions     []Extension `asn1:"optional"`
 }
+
+// OtherName represents the ASN.1 structure of the same name. See RFC
+// 5280, section 4.2.1.6.
+type OtherName struct{
```

Comment:
The ASN.1 tags of whatever the value is are in the Bytes field of the Value. The Value itself Will always parse out a Tag: 0, Class:2, IsCompound: true since the filed is actually an explicit [0] tag that wraps whatever the OtherName actually contains. I can't have it parse out the contents in detail because it can contain arbitrary data. There was no support for specific OtherName parsing added.

for reference:
OtherName ::= SEQUENCE {
        type-id    OBJECT IDENTIFIER,
        value      [0] EXPLICIT ANY DEFINED BY type-id }